decidim/decidim
# Remove etherpad-lite dependency

#### :tophat: What? Why?
Remove [etherpad-lite](https://github.com/jhollinger/ruby-etherpad-lite) dependency.

#### :pushpin: Related Issues
#8516
#7234

#### Testing
1. Download [etherpad server](https://github.com/ether/etherpad-lite)
2. Create APIKEY.txt to root directory of etherpad-lite server: ```echo CHANGE_ME_API_KEY > APIKEY.txt```
3. Run etherpad-lite server: ```./src/bin/run.sh```
4. Add etherpad server and api key to  ```.rbenv-vars```:
```
ETHERPAD_SERVER=http://localhost:9001
ETHERPAD_API_KEY=CHANGE_ME_API_KEY
```
5. Enable etherpad from ``development_app/config/initializers/decidim.rb``
6. Run development app: ``bundle exec rails s``
7. "Enable pads creation" in meetings component settings
8. Edit meeting so that it has started (but not ended)
9. Go to meeting's page and you should see etherpad in iframe.

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
![image](https://user-images.githubusercontent.com/19709320/143037638-8466de49-bd8c-4063-85d3-fc9254c4dfce.png)

:hearts: Thank you!
